setup/kubernetes/google.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Set Up a Google Kubernetes Engine Cluster
+description: Learn how to set up a GKE cluster for your Coder deployment.
+---
+
+This guide shows you how to set up a Google Kubernetes Engine (GKE) cluster to
+which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, make sure that the [gcloud
+CLI](https://cloud.google.com/sdk/docs/quickstarts) is installed on your
+machine and configured to interact with your Google Cloud Platform account.
+
+## Set Up the GKE Cluster
+
+The following will spin up a Kubernetes cluster using the `gcloud` command (be
+sure to replace the parameters (specifically `PROJECT_ID` and
+`NEW_CLUSTER_NAME`) as needed to reflect the needs of your environment).
+
+> If you're creating a cluster that supports use of the
+> [CVMs](../../admin/environment-management/cvms.md) deployment option to enable
+> users to run system-level programs like Docker or systemd, see the next sample
+> `gcloud` command.
+
+```bash
+gcloud beta container --project "PROJECT_ID" \
+clusters create "NEW_CLUSTER_NAME"
+   --zone "us-central1-a" \
+   --no-enable-basic-auth \
+   --cluster-version "1.14.7-gke.14" \
+   --machine-type "n1-standard-4" \
+   --image-type "COS" \
+   --disk-type "pd-standard" \
+   --disk-size "50" \
+   --metadata disable-legacy-endpoints=true \
+   --scopes "https://www.googleapis.com/auth/cloud-platform" \
+   --num-nodes "2" \
+   --enable-stackdriver-kubernetes \
+   --enable-ip-alias \
+   --network "projects/PROJECT_ID/global/networks/default" \
+   --subnetwork \
+   "projects/PROJECT_ID/regions/us-central1/subnetworks/default" \
+   --default-max-pods-per-node "110" \
+   --addons HorizontalPodAutoscaling,HttpLoadBalancing \
+   --enable-autoupgrade \
+   --enable-autorepair \
+   --enable-network-policy \
+   --enable-autoscaling
+   --min-nodes "1"
+   --max-nodes "8" \
+   --region "us-central1-c"
+```
```

Comment:
```suggestion
```bash
gcloud beta container --project "PROJECT_ID" \
clusters create "NEW_CLUSTER_NAME"
   --zone "us-central1-a" \
   --no-enable-basic-auth \
   --cluster-version "latest" \
   --machine-type "n1-standard-4" \
   --image-type "COS" \
   --disk-type "pd-standard" \
   --disk-size "50" \
   --metadata disable-legacy-endpoints=true \
   --scopes "https://www.googleapis.com/auth/cloud-platform" \
   --num-nodes "2" \
   --enable-stackdriver-kubernetes \
   --enable-ip-alias \
   --network "projects/PROJECT_ID/global/networks/default" \
   --subnetwork \
   "projects/PROJECT_ID/regions/us-central1/subnetworks/default" \
   --default-max-pods-per-node "110" \
   --addons HorizontalPodAutoscaling,HttpLoadBalancing \
   --enable-autoupgrade \
   --enable-autorepair \
   --enable-network-policy \
   --enable-autoscaling \
   --min-nodes "1" \
   --max-nodes "8"
```
```

@khorne3 did you get this command to work? I had to change a few things, such as missing backslashes and then I never was able to specify both a region and a zone, it was one or the other. Also the cluster version didn't work for me when I tried it as the current docs say, I changed it to `latest` instead.

Reply:
I updated to fix the region/zone thing and cluster version and to remove trailing whitespaces